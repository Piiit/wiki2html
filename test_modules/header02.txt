~~NOTOC~~
before
= header = 
after
== test == asdf asdf asdf asdf ==

asdf === test3 === 

**test asdf
multiline
**
