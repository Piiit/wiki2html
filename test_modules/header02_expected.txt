<!doctype html>
<html>
<head>
<title>wiki2html: no title</title>
</head>
<body>
before
<h1>header</h1>
after
<h2>test == asdf asdf asdf asdf</h2>

asdf === test3 ===

<b>test asdf
multiline
</b>
</body>
</html>

